tribe-ext-eventbrite-addl-opts/index.php

```diff
@@ -52,76 +46,63 @@ public function add_settings() {
+				'validation_type' => 'textarea',
 			),
 
-			// see https://central.tri.be/issues/62402
 			$this->opts_prefix . 'iframe_px' => array(
-				'type'				=> 'text',
-				'label'				=> esc_html__( 'Display Tickets iframe Height (pixels)', 'tribe-extension' ),
-				'default'			=> 200, // required else positive_int validation will throw an error each time settings are saved if this field is left blank
-				'tooltip'			=> esc_html__( 'The default is 200.', 'tribe-extension' ),
-				'validation_type'	=> 'positive_int',
+				'type'            => 'text',
+				'label'           => esc_html__( 'Display Tickets iframe Height (pixels)', 'tribe-extension' ),
+				'default'         => 200,
+				'tooltip'         => esc_html__( 'The default is 200.', 'tribe-extension' ),
+				'validation_type' => 'positive_int',
 			),
 
 			$this->opts_prefix . 'locale' => array(
-				'type'				=> 'dropdown',
-				'label'				=> esc_html__( 'Eventbrite Locale', 'tribe-extension' ),
-				'default'			=> 'en_US',
-				'tooltip'			=> sprintf( esc_html__( 'This affects the API call. The default is %sUnited States (en_US)%s.', 'tribe-extension' ), '<strong>', '</strong>' ),
-				'validation_type'	=> 'options',
-				'options'			=> $this->eb_locale_country_name_array(),
+				'type'            => 'dropdown',
+				'label'           => esc_html__( 'Eventbrite Locale', 'tribe-extension' ),
+				'default'         => 'en_US',
+				'tooltip'         => esc_html__( 'This affects the API call.', 'tribe-extension' ),
+				'validation_type' => 'options',
+				'options'         => $this->eb_locale_country_name_array(),
 			),
 
 			$this->opts_prefix . 'ticket_single_location' => array(
-				'type'				=> 'dropdown',
-				'label'				=> esc_html__( 'Single Event Display Location', 'tribe-extension' ),
-				'default'			=> 'tribe_events_single_event_after_the_meta',
-				'tooltip'			=> sprintf( esc_html__( 'Where should the Eventbrite Tickets iframe display on your Single Events pages? The default is %sAfter the Meta%s.', 'tribe-extension' ), '<strong>', '</strong>' ),
-				'validation_type'	=> 'options',
-				'options'			=> $this->single_event_display_locations_array(),
-			),
-
-			$this->opts_prefix . 'show_admin_cost_field' => array(
-				'type'				=> 'checkbox_bool',
-				'label'				=> esc_html__( 'Display The Events Calendar\'s "Event Cost" field.', 'tribe-extension' ),
-				'tooltip'			=> esc_html__( 'By default, The Events Calendar\'s "Event Cost" field is hidden when Eventbrite Tickets or Event Tickets Plus is active. Check this box to re-enable it.', 'tribe-extension' ),
-				'validation_type'	=> 'boolean',
+				'type'            => 'dropdown',
+				'label'           => esc_html__( 'Single Event Display Location', 'tribe-extension' ),
+				'default'         => 'tribe_events_single_event_after_the_meta',
+				'tooltip'         => esc_html__( 'Where should the tickets area display on your Single Events pages?', 'tribe-extension' ),
```

Comment:
In the interest of sticking with the same standards we use in the rest of our plugin, perhaps not asking a question in the tooltip makes sense. I'm not sure this tooltip is necessary, we don't have one in Event Tickets for the same form. But if you feel a tooltip is not too much, perhaps just reword it so that it's a helpful statement instead of a question.